src/query-ast-to-sql-ast/index.js

```diff
@@ -235,19 +235,20 @@ export function populateASTNode(
       aliasFrom += '@' + parentTypeNode.name
     }
     sqlASTNode.as = namespace.generate('column', aliasFrom)
-    // is it just a column? if they specified a sqlColumn or they didn't define a resolver, yeah
-  } else if (fieldConfig.sqlColumn || !field.resolve) {
+    // or maybe it just depends on some SQL columns
+  } else if (fieldConfig.sqlDeps) {
```

Comment:
Why move these around?

Reply:
Because fieldConfig.sqlDeps usually has field.resolve set and didn't match fieldConfig.sqlColumn || !field.resolve condition.
Now field.resolve is always set so fieldConfig.sqlDeps need higher priority.